src/CoralBenchmark.cc

```diff
@@ -5,20 +5,50 @@
 #include "utilsMpi.hh"
 #include "MC_Processor_Info.hh"
 
+#ifdef CORAL_2_BENCHMARK
```

Comment:
I think it will be better if we have no pre-processor flags for the benchmark check.  Code that isn't constantly compiled tends to rot and then fail when needed